packages/elements/src/components/ino-datepicker/ino-datepicker.tsx

```diff
@@ -157,7 +167,7 @@ export class Datepicker implements ComponentInterface {
 
   @Watch('inoDefaultDate')
   inoDefaultDateChanged(defaultDate: string) {
-    this.updateFlatpickr('inoDefaultDate', defaultDate);
+    this.flatpickr?.set('defaultDate', defaultDate);
```

Comment:
I suggest the former implementation doesn't work for setting 'inoDefaultDate' and 'inoDateFormat'. I think it would be great to add these fixes to the changelog.

Reply:
https://github.com/inovex/elements/issues/340

Reply:
I would suggest to fix this in a separate PR